apps/emqx_machine/src/emqx_machine.erl

```diff
@@ -47,6 +48,11 @@ start() ->
 graceful_shutdown() ->
     emqx_machine_terminator:graceful_wait().
 
+%% only used when failed to boot
+brutal_shutdown() ->
+    init:stop(),
+    timer:sleep(10_000).
```

Comment:
what's this for?

Reply:
If `init:stop/0` is called, does it reach this point?

Reply:
sorry. I have removed this sleep.
the pipe evaluation exits immediately after EOF anyway.